edn/src/pretty_print.rs

```diff
@@ -55,22 +53,22 @@ impl Value {
     /// readability and limited whitespace expansion.
     pub fn as_doc<'a, A>(&'a self, pp: &'a A) -> pretty::DocBuilder<'a, A>
         where A: pretty::DocAllocator<'a> {
-        match self {
-            &Value::Vector(ref vs) => self.bracket(pp, "[", vs, "]"),
-            &Value::List(ref vs) => self.bracket(pp, "(", vs, ")"),
-            &Value::Set(ref vs) => self.bracket(pp, "#{", vs, "}"),
-            &Value::Map(ref vs) => {
-                let xs = vs.iter().rev().map(|(ref k, ref v)| k.as_doc(pp).append(pp.space()).append(v.as_doc(pp)).group()).intersperse(pp.space());
+        match *self {
```

Comment:
Huh, I did not know there was a preference for `match *self ... Concrete::` vs. `match self ... &Concrete`.  I will start doing this, since presumably clippy has an _informed_ opinion.